python/cudf/cudf/tests/test_ops.py

```diff
@@ -45,10 +47,11 @@ def math_op_test(dtype, fn, nelem=128, test_df=False, positive_only=False):
     print(got)
     print("expect")
     print(expect)
```

Comment:
Should remove these print statements before merging

Reply:
Will do :D